pkg/pvresizing/pv.go

```diff
@@ -0,0 +1,27 @@
+package pvresizing
+
+const (
+	Name = "prometheus"
+
+	PrometheusStorageSizeSmall  PrometheusStorageSizeType = "small"
+	PrometheusStorageSizeMedium PrometheusStorageSizeType = "medium"
+	PrometheusStorageSizeLarge  PrometheusStorageSizeType = "large"
+)
+
+type PrometheusStorageSizeType string
+
+// PrometheusDiskSize returns the desired disk size based on the
+// value of annotation monitoring.giantswarm.io/prometheus-disk-size
```

Comment:
Can you not reference the annotation here? This is infrastructure level information that has no reason to be in the domain